### PR TITLE
[DOCFIX] Update renamed link in GCS

### PR DESCRIPTION
**Pull request by Xenorith**

### CODE REVIEW

**Comment by Xenorith** [comment]
@LuQQiu can you confirm that the link's redirect is still correct?

File: docs/en/ufs/GCS.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ When building your application to use Alluxio, your application should include a
 `alluxio-core-client-fs` module to use the
 [Alluxio file system interface]({{ '/en/api/FS-API.html' | relativize_url }}) or the
 `alluxio-core-client-hdfs` module to use the
-[Hadoop file system interface](https://wiki.apache.org/hadoop/HCFS). For example, if you
+[Hadoop file system interface](https://cwiki.apache.org/confluence/display/HADOOP2/HCFS). For example, if you
```
Comment (LuQQiu) [inline docs/en/ufs/GCS.md]: This docs looks really old, I suggest directly remove the whole `Configuring application dependency` section. They are not useful

**Comment by Xenorith** [comment]
doh, i thought you already reviewed this page for doc validation >.< so what to do with this PR?

**Comment by LuQQiu** [comment]
Oh.... that's the problem of thinking myself too familiar with the docs....
I verify the commands.... I will create a new PR to remove the whole section

**Comment by LuQQiu** [comment]
https://github.com/Alluxio/alluxio/pull/10271

**Pull request closed without merging**